actions/setup_and_build/action.yml

```diff
@@ -1,29 +1,25 @@
-
 name: 'Setup and build'
 description: 'Setup and build Session Desktop'
+inputs:
+  cache_suffix:
+    description: 'the package we are currently building (used as key for the cached node_modules)'
```

Comment:
Can you put an example suffix here please?

Reply:
not used by anything else than CIO

Reply:
Cool